app/move/controllers/create/prison-transfer-reason.test.js

```diff
@@ -0,0 +1,71 @@
+const fieldHelpers = require('../../../../common/helpers/field')
+const referenceDataService = require('../../../../common/services/reference-data')
+
+const FormController = require('hmpo-form-wizard').Controller
+
+const PrisonTransferReasonsController = require('./prison-transfer-reason')
+const prisonTransferReasonsController = new PrisonTransferReasonsController({
+  route: '/prison_transfer_reason',
+})
+
+describe('Prison transfer reason controller', function() {
+  let stubForMapReferenceDataToOption
+  let stubForGetPrisonTransferReasons
+  let stubForParentConfigure
+  let mockRequest
+
+  context('happy path', function() {
+    beforeEach(async function() {
+      mockRequest = {
+        form: {
+          options: {
+            fields: {
+              prison_transfer_reason: {},
+            },
+          },
+        },
+      }
+      stubForParentConfigure = sinon.stub(FormController.prototype, 'configure')
+      stubForMapReferenceDataToOption = sinon
+        .stub(fieldHelpers, 'mapReferenceDataToOption')
+        .returns({})
+      stubForGetPrisonTransferReasons = sinon
+        .stub(referenceDataService, 'getPrisonTransferReasons')
+        .resolves([{}, {}])
+      await prisonTransferReasonsController.configure(mockRequest, {}, () => {})
+    })
+    it('calls reference data service', function() {
+      expect(stubForGetPrisonTransferReasons).to.have.been.calledOnce
```

Comment:
You can actually avoid having to define the stub as a variable and just check the original method:

In beforeEach:
```javascript
...
sinon.stub(fieldHelpers, 'mapReferenceDataToOption').returns({})
...
```

In it block:

```javascript
expect(fieldHelpers.mapReferenceDataToOption).to.have.been.calledOnce
```

I find its a little easier to read too as you're not having to look for what the variables are.

Reply:
I am aware the variable is not strictly necessary, however I personally find it tidier this way. If that is a problem I am happy to change it.

Reply:
Well it was quicker to change it than to discuss it, so I changed it 😄